test/conformance/cts_exe.py

```diff
@@ -18,18 +20,24 @@
 
     parser = ArgumentParser()
     parser.add_argument("--test_command", help="Ctest test case")
-
     args = parser.parse_args()
+    
     result = subprocess.Popen([args.test_command, '--gtest_brief=1'], stdout = subprocess.PIPE, text = True)  # nosec B603
 
     pat = re.compile(r'\[( )*FAILED( )*\]')
-    for line in result.stdout.readlines():
+    output_list = []
+    for line in result.stdout:
+        output_list.append(line)
         if pat.search(line):
             test_case = line.split(" ")[5]
             test_case = test_case.rstrip(',')
             print(test_case)
 
-    result.communicate()
-    rc = result.returncode
+    rc = result.wait()
     if rc < 0:
-        print(signal.strsignal(abs(result.returncode)))
+        print(signal.strsignal(abs(rc)))
+
+    print("#### OUTPUT ####", file=sys.stderr)
```

Comment:
pls mark here, this output is from gtest

Reply:
Done